### PR TITLE
keras subclass embedding docfix

### PR DESCRIPTION
**Pull request by TimSchmeier**
see https://github.com/tensorflow/recommenders/issues/240

### CODE REVIEW

**Comment by TimSchmeier** [comment]
@googlebot I signed it!

**Comment by maciejkula** [comment]
Thank you!

**Comment by maciejkula** [comment]
We have some issues we need to fix on our end to merge this - we should be there early next week.

Thanks for testing our pipelines!

**Comment by TimSchmeier** [comment]
You bet. We actually met recently when you presented TFRS to Spotify. Super excited about the TPU work. Cheers

**Review by gagika — APPROVED** [review]
Thanks

File: tensorflow_recommenders/layers/embedding/tpu_embedding_layer.py [hunk 1]
```diff
@@ -330,18 +330,18 @@ class instance you specify will be used to manage the slot variables. This
...
-        'feature_three': tf.keras.Input(batch_size=1024, shape=(),
+        'feature_three': tf.keras.Input(batch_size=1024, shape=(1,),
                                         dtype=tf.int32)}
     # embedding, feature_config and embedding_inputs all have the same nested
     # structure.
     embedding = tpu_embedding_layer.TPUEmbedding(
         feature_config=feature_config,
         optimizer=tf.tpu.experimental.embedding.SGD(0.1))(embedding_inputs)
-    logits = tf.keras.layers.Dense(1)(tf.concat(tf.nest.flatten(embedding)))
+    logits = tf.keras.layers.Dense(1)(tf.concat(tf.nest.flatten(embedding), axis=1))
```
Comment (gagika) [inline tensorflow_recommenders/layers/embedding/tpu_embedding_layer.py:344]: Could you please make sure that this line doesn't go over 80 characters?

Comment (TimSchmeier) [reply tensorflow_recommenders/layers/embedding/tpu_embedding_layer.py:344]: yes will do.

**Pull request merged**